pkg/ddl/partition.go

```diff
@@ -3230,7 +3230,7 @@ func (w *worker) onReorganizePartition(jobCtx *jobContext, t *meta.Meta, job *mo
 		// and we will soon start writing to the new partitions.
 		if s, ok := jobCtx.store.(kv.SplittableStore); ok && s != nil {
 			// partInfo only contains the AddingPartitions
-			splitPartitionTableRegion(w.sess.Context, s, tblInfo, partInfo.Definitions, true)
+			splitPartitionTableRegion(w.sess.Context, s, tblInfo, partInfo.Definitions, scatterTable)
```

Comment:
Why here use "table" level scatter stratage?

Reply:
1. The DDL statements that trigger split region need to wait for the scatter region to complete. 
2. yes, 'on' equal 'scatterTable' here, i will add some comment.